src/CommunityToolkit/Xamarin.CommunityToolkit.Sample/Pages/Views/SnackBarPage.xaml.cs

```diff
@@ -35,9 +35,8 @@ async void DisplaySnackBarAdvancedClicked(object sender, EventArgs args)
 			var messageOptions = new MessageOptions
 			{
 				Foreground = Color.DeepSkyBlue,
-				FontSize = 40,
-				FontFamily = "Times New Roman",
-				Message = GenerateLongText(5)
+				Font = Font.OfSize("FARegular", 40),
+				Message = "\uf2bb"
```

Comment:
I propose to convert font to class with something like https://andreinitescu.github.io/IconFont2Code/
And use constants instead :)

Reply:
I added this code just for example. Do you think it is worth to add all icons? Maybe create a constant just for this text?

Reply:
Would be enough it, agreed 👍